cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -899,10 +899,14 @@ function Start-WorkerServices {
   $kubelet_args = $kubelet_args_str.Split(" ")
   Log-Output "kubelet_args from metadata: ${kubelet_args}"
 
-  $additional_arg_list = @(`
+  $kubeproxy_args_str = ${kube_env}['KUBEPROXY_ARGS']
+  $kubeproxy_args = $kubelet_args_str.Split(" ")
```

Comment:
This should be kubeproxy_args_str, right?

Reply:
Thanks! Done.